Authentication.md

```diff
@@ -619,6 +619,100 @@ if err != nil {
 ```
 
 
+## Multi-Cloud Saas Platform (MCSP) Authentication
+The `MCSPAuthenticator` can be used in scenarios where an application needs to
+interact with an IBM Cloud service that has been deployed to a non-IBM Cloud environment (e.g. AWS).
+It will accept a user-supplied apikey and will perform the necessary interactions with the
+Multi-Cloud Saas Platform token service to obtain a suitable MCSP bearer token for the specified apikey.
+The authenticator will also obtain a new bearer token when the current token expires.
+The bearer token is then added to each outbound request in the `Authorization` header in the
+form:
+```
+   Authorization: Bearer <bearer-token>
+```
+
+### Properties
+
+- ApiKey: (required) the IAM apikey to be used to obtain an IAM access token.
+One of ApiKey or RefreshToken must be specified.
```

Comment:
Is this a copy/paste error? I don't see `RefreshToken` in the list of properties

Reply:
fixed